package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "bower": "node_modules/bower/bin/bower update --allow-root",
     "themes": "node vendor/claroline/distribution/main/core/Resources/scripts/build-themes",
+    "old-themes": "node --use_strict vendor/claroline/distribution/main/core/Resources/scripts/build-old-themes",
```

Comment:
I struggle to find any official doc on that `--use-strict` option but it seems it will put every module in strict mode, including third-party modules required by your script. That could be a problem at some point. Using the `'use strict'` directive would probably be safer.